src/components/Releases.js

```diff
@@ -151,7 +151,7 @@ const communityReleases = [
                 url: 'https://projects.eclipse.org/projects/ecd.cdt-cloud',
                 version: '0.0.108',
```

Comment:
I think we should change the version label here too. Thank you!

Reply:
Yes! I added a new commit. Can you squash them when you merge please.